cnxarchive/views/content.py

```diff
@@ -218,8 +218,11 @@ def get_content_json(request):
     """Retrieve content as JSON using the ident-hash (uuid@version)."""
     result = _get_content_json()
 
-    result = json.dumps(result)
     resp = request.response
+    if result['baked'] is not None:
+        resp.cache_control = "no-cache"
```

Comment:
You're right this logic is backwards. We want to `no-cache` when not baked. Note that `baked` will come across as a `datetime.datetime` object, when set, so the correct test is in fact for `is None`